py-scripts/lf_ftp.py

```diff
@@ -2238,8 +2626,27 @@ def pass_fail_duration(band, file_size):
                               dowebgui=args.dowebgui,
                               device_list=args.device_list,
                               test_name=args.test_name,
+                              eap_method=args.eap_method,
+                              eap_identity=args.eap_identity,
+                              ieee80211=args.ieee8021x,
```

Comment:
Let's just fix this later, but please also update the class in initializer when adjusting new arguments.